CRITICAL - 2022-03-12 19:45:36 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-03-12 19:47:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 19:48:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 19:48:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 19:50:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 19:51:05 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 19:59:24 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 19:59:30 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:03:23 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#5 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(28): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->create()
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-03-12 20:04:50 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:04:59 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:05:15 --> Too few arguments to function App\Controllers\Pegawai::update(), 0 passed in C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php on line 832 and exactly 1 expected
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->update()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-03-12 20:11:05 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:11:30 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#5 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(28): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->create()
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-03-12 20:11:37 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:13:36 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:14:28 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:24:53 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:25:36 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:25:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:25:59 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:27:13 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:30:14 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:30:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:30:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:30:21 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:30:23 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 20:38:03 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Pegawai.php(8): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Pegawai->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 21:13:57 --> Table 'user.pegawai' doesn't exist
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 21:18:59 --> Table 'user.pegawai' doesn't exist
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 21:19:52 --> Table 'user.pegawai' doesn't exist
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 21:21:04 --> Unknown column 'nip' in 'field list'
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 21:23:24 --> Unknown column 'nip' in 'field list'
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 21:25:12 --> Column 'nip' cannot be null
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 21:25:37 --> Column 'nip' cannot be null
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 22:37:47 --> Column 'nip' cannot be null
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 22:39:14 --> Column 'nip' cannot be null
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 22:42:53 --> Undefined variable: pegawai
#0 C:\xampp\htdocs\latihan\app\Views\mahasiswa\index.php(111): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 111, Array)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('mahasiswa/index', Array, true)
#4 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(13): view('mahasiswa/index', Array)
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 22:44:34 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(10): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 22:44:54 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(10): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 22:45:26 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(10): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->index()
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-03-12 23:08:31 --> Table 'user.pegawai' doesn't exist in engine
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 23:12:14 --> Table 'user.pegawai' doesn't exist in engine
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 23:16:22 --> Table 'user.pegawai' doesn't exist in engine
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 23:17:11 --> Table 'user.pegawai' doesn't exist in engine
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 23:47:47 --> Table 'user.pegawai' doesn't exist in engine
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pe...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pe...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pe...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->create()
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#10 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-12 23:51:06 --> Table 'user.pegawai' doesn't exist in engine
#0 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\latihan\app\Controllers\Mahasiswa.php(10): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Mahasiswa->index()
#8 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Mahasiswa))
#9 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\latihan\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\latihan\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
